ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/remove_container/tasks/main.yml

```diff
@@ -0,0 +1,24 @@
+---
+# The deleteNodes.py script deletes the nodes in Jenkins
+# Ansible then removes the docker containers on the dockerhost via the delete_container role
+# If the node is not idle in Jenkins, the node is not deleted
+
+# no_log hides the api_token from the output
+- name: Run deleteNodes script
+  script: scripts/deleteNodes.py {{ jenkins_username }} {{ jenkins_api_token }} {{ delete_nodes }}
```

Comment:
I think I'm missing a little bit of higher level context here - where is `{{ delete_nodes }}` coming from i.e. how is it selecting what to delete? This seems to be the only reference to that variable in this PR so it possibly needs a bit of clarification

Reply:
Im working on updating the docs to explain all of this. {{ delete_nodes }} will be passed to the playbook as an ansible extra variable